recipes/LibriSpeech/self-supervised-learning/BEST-RQ/hparams/BEST-RQ.yaml

```diff
@@ -35,7 +35,7 @@ number_of_epochs: 3000
 optimizer_step_limit: 300000
 
 # This setup is for 8 V100.
-seconds_per_batch: 400
+max_batch_len: 400 # In second and per gpu
 train_num_buckets: 150
```

Comment:
Are the BEST-RQ recipe changes related to the DDP fixes?

Reply:
Not really... but they are here on purpose -- these are minor "cosmetic" changes that I don't want to do a PR for..